pkg/apis/wildfly/v1alpha2/wildflyserver_types.go

```diff
@@ -0,0 +1,151 @@
+package v1alpha2
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// WildFlyServerSpec defines the desired state of WildFlyServer
+// +k8s:openapi-gen=true
+type WildFlyServerSpec struct {
```

Comment:
We would have to find a convention in the comments to explains the difference to the previous version. It's difficult to figure that out when looking at the whole struct.

Reply:
maybe there is an annotation similar to the javadoc `@since`